client/ayon_usd/standalone/usd/pinning/__pinning_file_generation_funcs.py

```diff
@@ -0,0 +1,362 @@
+import logging
+import json
+import os
+import sys
+import re
+from typing import Dict, List, Optional, Set
+from pxr import UsdShade, Ar, Sdf
+from urllib.parse import urlparse
+
+log = logging.getLogger(__name__)
+
+def _normalize_path(path):
+    if sys.platform.startswith('win'):
+        if len(path) > 1 and path[1] == ':':
+            path = path[0].lower() + path[1:]
```

Comment:
Requires `import platform`.
```suggestion
    if (
        platform.system().lower() == "windows"
        and len(path) > 1
        and path[1] == ":"
    ):
        path = path[0].lower() + path[1:]
```

Reply:
changed in external commit. 
still using sys.platform as we already import sys and importing platform would be an extra import. 

new version. 
```py
def _normalize_path(path):
    if (sys.platform.startswith('win')
        and len(path) > 1 
        and path[1] == ':'  
        ):
        
        path = path[0].lower() + path[1:]
    
    return os.path.normpath(path)

```